include/flexflow/ops/cast.h

```diff
@@ -13,22 +13,13 @@
  * limitations under the License.
  */
 #pragma once
```

Comment:
Change to ifdef for consistency (yeah I know you didn't make this change, but just throw it in this PR :hearts:)